### ISSUE TITLE
[data-background-iframe] doesn't support URL queries

### ISSUE THREAD

**Issue by ulfben** [OleVik/grav-plugin-presentation#5]
I'm trying to embed a full-screen background iframe, and need to pass a bunch of variables to the page in the iframe.  But when providing a URL with a query string to the shortcode nothing after "?" is rendered in the front-end.

**Examples:**
`[data-background-iframe=https://example.net]` works perfectly and outputs 
`<section [...] data-background-iframe="https://example.net">`

But add some variables: 
`[data-background-iframe=https://example.net?h=f&sid=1006&p=0.m]` and you get: 
`<section [...] data-background-iframe="https://example.net?">`

Manually escape the variables:  
`[data-background-iframe=https://example.net?h=f&amp;sid=1006&amp;p=0.m]` and you still get: 
`<section [...] data-background-iframe="https://example.net?">`

I've tried wrapping the URL in quotes but that breaks the short code entirely. 

Manually writing an HTML iframe works fine with the query so it seems to be specific to the shortcode handling of URLs?

**Comment by OleVik** [comment]
Yes, [the regex](https://github.com/OleVik/grav-plugin-presentation/blob/v1.6.4/API/Parser.php#L36) [does not handle parameters correctly](https://regex101.com/r/oFyhTP/1). There is no quick fix for this, however, so for the time being you'll have to rely on plain HTML.

**Comment by OleVik** [comment]
This should be fixed with [v1.7.0-beta.1](https://github.com/OleVik/grav-plugin-presentation/releases/tag/v1.7.0-beta.1). Use double quotes around the value of the shortcode and the interactive-shortcode to elevate it to the foreground:

```
[data-background-iframe="https://example.net?h=f&sid=1006&p=0.m"]
[data-background-interactive]
```

Please test and confirm whether this works.

**Comment by ulfben** [comment]
Confirmed! The background iframe now accepts query parameters and seem to work as intended. 

Great job, and thanks!
